src/playbooks/checks/check.yaml

```
@@ -0,0 +1,9 @@
+---
+- name: Run preflight checks
+  hosts: localhost
```

This should be on the quadlet host because that's where we run podman.
```suggestion
  hosts: quadlet
```